rega-genotype/src/main/java/rega/genotype/GenotypeTool.java

```diff
@@ -60,7 +61,7 @@ private static ArgsParseResult parseArgs(String[] args) {
         CmdLineParser.Option treeGraphCmdOption = parser.addStringOption('g', "treegraph");
         
         try {
-            parser.parse(args);
+        	parser.parse(args);
```

Comment:
Please remove all the changes to the whitespaces that have no effect on this file